packages/server/database/migrations/20200610125717-cleanUsers.ts

```diff
@@ -0,0 +1,25 @@
+import {R} from 'rethinkdb-ts'
+
+export const up = async function(r: R) {
+  try {
+    await r
+      .table('User')
+      .replace((row) =>
+        row.without(
+          'welcomeSentAt',
+          'cacheExpiresAt',
+          'cachedAt',
+          'emailVerified',
+          'name',
+          'nickname'
+        )
+      )
+      .run()
+  } catch (e) {
+    console.log(e)
+  }
+}
+
+export const down = async function() {
+  // noop
```

Comment:
Why do we not provide a `down` method?

Reply:
the `up` method removes fields from the `User` object. once deleted, we can't restore them, e.g. when we remove the `name` field, there's no way we can add the `name` field back since we don't know what it was.

Reply:
Gotcha! Thanks for the explanation.